Server/scatterplotProtocol.py

```diff
@@ -16,26 +16,104 @@
 # data, math
 import vtk, vtk.util.numpy_support
 # from vtk.util.numpy_support import vtk_to_numpy, numpy_to_vtk
-# from vtkPVServerManagerRenderingPython import vtkSMPVRepresentationProxy
+from vtkPVServerManagerRenderingPython import vtkSMPVRepresentationProxy
```

Comment:
Do you really need that? It fails on my mac and not having it just works.
Maybe we want to have it in a try/catch if it is really needed

Reply:
`vtkSMPVRepresentationProxy` is used to update the color maps. That's why you are seeing white points, probably. Why would it fail? You could run ERS on your mac with this line, right?

Reply:
I don't know. The script crash at the import for me.

Reply:
it doesn't find vtkPVServerManagerRenderingPython